cypress/utils/commands.js

```diff
@@ -676,3 +740,28 @@ Cypress.Commands.add('removeSampleDataFromDataSource', (dataSourceTitle) => {
     .should('be.visible')
     .click();
 });
+
+Cypress.Commands.add('createDataSourceNoAuthWithTitle', (title) => {
```

Comment:
is this really needed? We already have cypress command to create data source from API, which will be simpler and faster than this. We can modify the existing createDataSource cypress command to take title as a parameter

Reply:
Hi, creating data source from api will not update uiSettings, for console default testing, only creating data source from UI will update the default data source, api create will not talk to uiSettings, therefore, have a UI creation separately